packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1056,7 +1063,7 @@ class Auth {
     async loginIDPUser(parameters: LoginIDPUserParams) {
         const codeVerifier = this.get('code_verifier')
         const code = parameters.code
-        const usid = parameters.usid
+        const usid = parameters.usid || this.get('usid')
```

Comment:
Under what circumstances would the usid on parameters be undefined?

Reply:
Is usid on LoginIDPUserParams type optional? .. seems a little odd that it would be.

Reply:
Yes it's optional... If it's not provided then a new usid is generated...